cmd/snap-confine/snap-confine.apparmor.in

```diff
@@ -554,16 +554,21 @@
         # in /tmp/.snap/ during the preparation of a writable mimic.
         # FIXME: update this with per-snap snap-update-ns profiles
         mount options=(bind, rw) /** -> /tmp/.snap/**,
+        mount options=(rbind, rw) /** -> /tmp/.snap/**,
         # Allow mounting tmpfs over the original read-only directory.
         # FIXME: update this with per-snap snap-update-ns profiles
         mount fstype=tmpfs options=(rw) tmpfs -> /**,
         # Allow bind mounting anything from the temporary place in /tmp/.snap/
         # back to $SNAP/** (to re-construct the data that was there before).
         # FIXME: update this with per-snap snap-update-ns profiles
         mount options=(bind, rw) /tmp/.snap/** -> /**,
+        mount options=(rbind, rw) /tmp/.snap/** -> /**,
         # Allow unmounting the temporary directory in /tmp once it is no longer
         # necessary.
         umount /tmp/.snap/**,
+        # Allow unmounting any of the above in case something fails and
+        # we start recovery.
+        umount /**,
```

Comment:
I guess this will be updated with per-snap-update-ns profiles? Let's please make sure that happens for 2.32. With this rule, the rule above it is no longer needed.

Reply:
Yes